examples/2.usdc_balance.ts

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
I do not think we need to put any example for showing user how to query your usdc balance from usdc contract